### PR TITLE
AddIing Reducers and Actions

### PR DESCRIPTION
**Pull request by Kidd254**
In this pull request I:
- [ ] Added Redux Toolkit (npm install react-redux @reduxjs/toolkit)
- [ ] Created a directory that will contain all my Redux logic (/src/redux)
- [ ] Configure da Redux store (/src/redux/store.js)
- [ ] Defined a slice of state for books that:
1.     Includes an array of books (initial state: empty array)
2.     Includes a reducer that adds a book
3.     Includes a reducer that removes a book
- [ ] Defined a slice of state for categories that:
1.     Includes an array of categories (initial state: empty array)
2.     Includes a reducer that checks the status and always returns "Under construction" 
- [ ] Structured my application files using a "feature folder" approach and used the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for my Redux files.

### CODE REVIEW

File: src/Router.js
Comment (LionRouge1) [inline src/Router.js:1]: - [ ] Kindly do not push empty files to your codebase. Simply remove the file from your project if you do not need it. That will make your project look more professional and clean.

File: src/reportWebVitals.js
Comment (LionRouge1) [inline src/reportWebVitals.js:1]: - [ ] Please remove all the default files created by the `npx create-react-app` react command to make your project clean and YAGNI

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Kidd254, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job configuring redux for books and categories :+1: 
- Nice work use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [ ] This milestone is the suite of the "Bookstore" app which you started building in the previous activities. But there is no sign of the components and navigation bar created in the previous activities in your App. Kindly make sure your application files/folders structure looks like this :point_down: to follow the project requirement 
```sh
/src
    |---/components
    |---/redux
        |--- /books
                | booksSlice.js
        |--- /categories
                | categoriesSlice.js
        | store.js
    | index.js
```
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @Kidd254  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Previous react components works fine💯
-  Setup redux store correrctly👍
-  Followed ducks pattern🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**